stable/datadog/values.yaml

```diff
@@ -360,6 +360,25 @@ clusterAgent:
   #
   # readinessProbe:
 
+  ## @param service - object - optional
+  ##
+  # serviceType: ClusterIP
+  ##
+  #
+  service:
+    type: ClusterIP
+    annotations: {}
```

Comment:
why did you add `annotations: {}`?
I didn't see where it is used.

Reply:
you are correct, that's a mistake. I have removed it.